.github/workflows/_test-units.yml

```diff
@@ -0,0 +1,57 @@
+name: Test Units
+
+on:
+  workflow_call:
+    inputs:
+      node-version:
+        description: 'Version of node to use'
+        required: false
+        type: number
+        default: 20
+
+concurrency:
+  group: test-units-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+
+  run:
+    runs-on: ubuntu-latest
+
+    strategy:
+        matrix:
+          theme: [default, bootstrap, material, classic, fluent]
+
+    steps:
+
+      - name: Checkout branch
+        uses: actions/checkout@v4
+
+      - name: Setup node
+        id: setup-node
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ inputs.node-version }}
+
+      - name: Use cache for root node_modules
+        id: cache-root-node_modules
+        uses: actions/cache@v4
+        with:
+          path: node_modules
+          key: root-node_modules-${{ steps.setup-node.outputs.node-version }}-${{ hashFiles('package-lock.json') }}
+
+      - name: Install
+        if: steps.cache-root-node_modules.outputs.cache-hit != 'true'
+        run: |
+          npm ci --no-audit --no-fund
+
+      - name: Build test assets
+        run: |
+          npm run sass --prefix packages/core
```

Comment:
Wouldn't this also would if we call `npm run docs` just once after the themes are compiled?